packages/cozy-client/src/models/country/countries.js

```diff
@@ -433,3 +276,68 @@ export const checkCountryCode = code => {
   )
   return !!foundCountry
 }
+
+/**
+ * Get country name by its ISO 3166-1 (alpha-2, alpha-3 or numeric code)
+ *
+ * @param {string|number} code - ISO 3166-1 alpha-2, alpha-3 or numeric code
+ * @param {object} options - Options
+ * @param {string} options.lang - Language of country names
+ * @returns {string|number} - Country name or code if not found
```

Comment:
Looks like it actually returns null if not found, not code (which is good I think)